ext/web/09_file.js

```diff
@@ -653,6 +654,33 @@ function blobFromObjectUrl(url) {
   return blob;
 }
 
+/**
+ * @param {Blob} blob
+ * @returns {string}
+ */
+function createObjectURL(blob) {
```

Comment:
Moved these two functions from `11_blob_url.js` which is not explicitly imported by any file.